src/checkboxes.ts

```diff
@@ -0,0 +1,203 @@
+'use strict';
+
+import { window, workspace, TextLine, Range, TextEditor, TextEditorEdit } from 'vscode';
+
+// Checkbox is represented by exactly one symbol between square brackets.  Symbol indicates status: '-' undetermined, 'x' or 'X' checked, ' ' unchecked.
+const checkboxRegex = /\[([-xX ])\]/;
+// Summary is a cookie indicating the number of ticked checkboxes in the child list relative to the total number of checkboxes in the list. 
+const summaryRegex = /\[(\d*\/\d*)\]/;
+// Percentage is a cookie indicating the percentage of ticked checkboxes in the child list relative to the total number of checkboxes in the list. 
+const percentRegex = /\[(\d*)%\]/;
+const indentRegex = /^(\s*)\S/;
+let orgTabSize: number = 4;
+    
+export function OrgTabsToSpaces(tabs: string, tabSize: number = 4): number {
```

Comment:
Hi @aboukirev, in JS/TS functions are usually named in lower camel case style, even if they're exported.

Reply:
Agreed.  Thanks for reviewing.  Fixing it.